### DISCUSSION TITLE
현재 라이팅 문제가 있습니다

### DISCUSSION THREAD

**Discussion by ted002** [naverz/zepeto-studio-kor#515] [5. Q&A]
![문제 1](https://user-images.githubusercontent.com/73985764/159396066-84976cf0-ef34-44f1-989b-2cab10049934.JPG)

![KakaoTalk_20220322_113052445](https://user-images.githubusercontent.com/73985764/159396091-bfc5c202-e832-4730-9c38-7f184f3e410b.jpg)

보시는것처럼 유니티에서의 색이 제패토 앱을 실행시켜서 확인하면 색조가 이상하게 나옵니다. 라이팅 문제인듯 합니다.

https://user-images.githubusercontent.com/73985764/159396175-1f76d41d-fa4f-4b3a-9ec1-ab976880bb50.mp4

그리고 해당 영상에 나오는것처럼 유니티에서 씬 이동시에 색조가 이상하게 나오는 경우도 있습니다.
라이팅의 문제같은데, 어떻게 해결방법이 없나요?

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 해당 문제 확인해 보겠습니다.

**Comment by Zepeto-TechEvangelist** [comment]
### [에디터와 앱의 차이 해결 방법]

Edit> Project Setting > Quality에서 PC와 iOS, Android 퀄리티 차이가 다른게 기본이라서
퀄리티 차이가 날 수 있습니다.
퀄리티를 동일하게 맞춰주세요~

<img width="631" alt="스크린샷 2022-03-23 오후 6 28 47" src="https://user-images.githubusercontent.com/100118650/159668502-e4a7f465-474c-418a-a22a-be03ca0a8af0.png">


### [ 씬 이동시 라이팅이 사라지는 문제 ]

이 문제 확인을 위해서 Window > Rendering > Lighting > Environment 탭을 누른 후
속성 값들이 잘 보이도록 캡쳐해서 올려주세요. 

<img width="726" alt="스크린샷 2022-03-23 오후 6 35 08" src="https://user-images.githubusercontent.com/100118650/159668580-d26104f3-8e2a-4892-a8df-19d66b7425e5.png">

**Reply by ted002** [reply]
![질문용](https://user-images.githubusercontent.com/73985764/159818911-c470f6c3-29fa-446d-ad01-00e800040c74.JPG)
이렇게 올리면 될까요?
그리고 상단에 말씀해주신 방법을 다 써봤는데 여전히 동일한 문제가 발생합니다

**Reply by Zepeto-TechEvangelist** [reply]
네 올려주신 내용 바탕으로 한번 더 확인해 보겠습니다.
